Endpoint.swift

```diff
@@ -8,45 +8,13 @@
 
 import Foundation
 
-
 enum EndpointMethod: String {
     case get = "GET"
     case post = "POST"
     case delete = "DELETE"
 }
 
-/// Looks at Secrets/Keys.plist file which should contain as key:
-/// - "api-url" should be the host of the deployed backend url
-enum Keys: String {
-    case apiURL = "api-url"
-    case apiDevURL = "api-dev-url"
-
-    var value: String {
-        switch self {
-        case .apiDevURL: return "localhost"
-        case .apiURL: return Keys.keyDict[rawValue] as! String
-        }
-    }
-
-    static var hostURL: Keys {
-        #if DEBUG
-        return Keys.apiDevURL
-        #else
-        return Keys.apiURL
-        #endif
-    }
-
-    private static let keyDict: NSDictionary = {
-        guard let path = Bundle.main.path(forResource: "Keys", ofType: "plist"),
-            let dict = NSDictionary(contentsOfFile: path) else { return [:] }
-        return dict
-    }()
-}
-
-
 struct Endpoint {
-    static var apiVersion: Int?
```

Comment:
Do we want to remove this? I'm pretty sure all of our backends are versioned. Just worried that when people try to integrate this into their project, they might forget

Reply:
I know all of ours do, but not everyone's. I guess I was looking more into public open source usability (specifically for the way I'm going to use it in our new AppDev demo app)

Reply:
ah ok makes sense